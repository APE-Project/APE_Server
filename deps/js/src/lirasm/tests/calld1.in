a = immd 1.1;
b = immd 2.2
c = immd 3.3;
d = immd 4.4;
e = immd 5.5;
f = immd 6.6;
g = immd 7.7;
h = immd 8.8;

res = calld calld1 cdecl a b c d e f g h
retd res
