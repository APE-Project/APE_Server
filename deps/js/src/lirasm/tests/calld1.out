Output is: -62.9667
